apps/api/src/app/gene/gene.controller.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  Controller,
+  Get,
+} from '@nestjs/common';
+import { GeneService } from './gene.service';
+import {ModuleRef} from "@nestjs/core";
+
+
+@Controller('genes')
+export class GeneController {
+  constructor(private readonly geneService: GeneService) {}
+
+    @Get()
+  async getHugoGenes() {
+    try {
+      const symbols = await this.geneService.fetchGeneSymbols();
+      return symbols;
```

Comment:
```suggestion
      return await this.geneService.fetchGeneSymbols();
```

Reply:
made this change and tested that the api and dropdown both still work